app/serializers/activitypub/add_hashtag_serializer.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ActivityPub::AddHashtagSerializer < ActivityPub::Serializer
```

Comment:
I'm not sure if this serializer has to be separate if the only difference is the type of `proper_object` we should be able to use a single `AddSerializer` for everything, see e.g. `Collection` and `Activity`